src/sql/base/browser/ui/panel/panel.ts

```diff
@@ -143,7 +144,7 @@ export class TabbedPanel extends Disposable {
 		tabLabel.innerText = tab.title;
 		tabElement.appendChild(tabLabel);
 		tab.disposables.push(DOM.addDisposableListener(tabHeaderElement, DOM.EventType.CLICK, e => this.showTab(tab.identifier)));
-		tab.disposables.push(DOM.addDisposableListener(tabHeaderElement, DOM.EventType.KEY_DOWN, (e: KeyboardEvent) => {
+		tab.disposables.push(DOM.addDisposableListener(tabHeaderElement, DOM.EventType.KEY_UP, (e: KeyboardEvent) => {
```

Comment:
This change to keep in line with the tree components which by default does enter operations on key up